src/Stryker.Core/Stryker.Core.UnitTest/Compiling/CompilingProcessTests.cs

```diff
@@ -239,5 +238,58 @@ public int Subtract(int first, int second)
                 ms.Length.ShouldBeGreaterThan(100, "No value was written to the MemoryStream by the compiler");
             }
         }
+
+        [Fact]
+        public void CompilingProcessTests_MustIncludeVersionInfo()
+        {
+            var syntaxTree = CSharpSyntaxTree.ParseText(@"using System;
+
+namespace ExampleProject
+{
+    public class Calculator
+    {
+        public int Subtract(int first, int second)
+        {
+            return first - second;
+        }
+    }
+}");
+            var input = new MutationTestInput()
+            {
+                ProjectInfo = new ProjectInfo()
+                {
+                    ProjectUnderTestAnalyzerResult = new ProjectAnalyzerResult(null, null)
+                    {
+                        Properties = new Dictionary<string, string>()
+                        {
+                            { "AssemblyTitle", "AssemblyName"},
+                        },
+                        Resources = new List<ResourceDescription>()
+                    },
+                    TestProjectAnalyzerResult = new ProjectAnalyzerResult(null, null)
+                    {
+                        Properties = new Dictionary<string, string>()
+                        {
+                            { "AssemblyTitle", "AssemblyName"},
+                        },
+                        Resources = new List<ResourceDescription>()
+                    }
+                },
+                AssemblyReferences = new List<PortableExecutableReference>() {
+                    MetadataReference.CreateFromFile(typeof(object).Assembly.Location)
+                }
+            };
+            var rollbackProcessMock = new Mock<IRollbackProcess>(MockBehavior.Strict);
+
+            var target = new CompilingProcess(input, rollbackProcessMock.Object);
+
+            using (var ms = new MemoryStream())
+            {
+                var result = target.Compile(new Collection<SyntaxTree>() { syntaxTree }, ms, false);
+                result.Success.ShouldBe(true);
+
+                Assembly.Load(ms.ToArray()).GetName().Version.ToString().ShouldBe("0.0.0.0");
```

Comment:
Should we not assert for GetVersionInfo as well? Since that is the example we saw in VFSForGit

https://github.com/Microsoft/VFSForGit/blob/25d21da9e286eef33db635833a92cad0055923da/GVFS/GVFS.Common/ProcessHelper.cs#L46

Reply:
Yes, but we can't because to do so we have to write to disk as FileVersionInfo.GetVersionInfo only accepts a string path and thus cannot use our mock filesystem